Tensile/KernelWriterAssembly.py

```diff
@@ -3979,6 +4011,29 @@ def graFinalOffsets(self, kernel, tP):
     tileOffsets = tP["vgprTileOffsets"]
     unrollOffsets = tP["gpr"]["unrollOffsets"]
     tmp = self.vgprPool.checkOut(3, self.preventVgprOverflowDuringNewTile)
+    waveStartSgpr = self.getTmpSgpr(1)
+    if not tP["tlu"]:
+      if kernel["TransposeLDS"]:
+        # add wave starting offset to Offset0I(A), Offset1J(B) registers
+        if tP["grcg"]:
+          if tP["grcv"]:
+            divisorName = tP["lvc"]
+          else:
+            # Fractional load use the more accurate lsc, multiply by VW later
+            divisorName = tP["lsc"]
+        else:
+          if tP["grcv"]:
+            divisorName = tP["lsp"]
+          else:
+            divisorName = tP["lvp"]
+        numPerpElementsPerLoad = kernel[divisorName]
+        numPerpElementsPerWave = tP["nrp"]*numPerpElementsPerLoad
```

Comment:
should be 
numPerpElementsPerWave = tP["nrp"] * globalParameters["WavefrontWidth"] // numPerpElementsPerLoad
kernel['LVCA'] mean how many thread load in unroll dim.
"globalParameters["WavefrontWidth"] // kernel['LVCA']" mean how many M lines can be loaded for 1 Wavefront in one buffer load instruction.
(tP["nrp"] * globalParameters["WavefrontWidth"] // numPerpElementsPerLoad) mean how many M lines can be loaded for 1 Wavefront.

Reply:
had fix in directToLds , fix is to use LSP/LVP parameters to calculate number of PerpelementsPerWave